src/main/kotlin/com/zepben/ewb/database/sqlite/cim/network/NetworkDatabaseReader.kt

```diff
@@ -57,7 +57,7 @@ class NetworkDatabaseReader internal constructor(
     databaseDescription,
     databaseTables,
     createMetadataReader,
-    createServiceReader
+    createServiceReader,
```

Comment:
chore: prefer either leaving files unmodified where possible (clean history), or if you decide to fix a pattern in a file that you change all instance in that file (code improvement). In this case, can you either revert this change, of add the trailing common to the other "last multi-line arguments" in this file.

Reply:
i think i did the thing